src/prplmesh_set.lua

```diff
@@ -68,6 +68,151 @@
 #
 ################################################################################
 --]]
+
 require("mmx/ing_utils")
+require("prplmesh-be-utils")
+
+
+--[[
+    @brief  Function for converting CMD arguments to key-value table.
+
+    @param  args Command Line arguments.
+            Example: Controller.Network.AccessPoint 1 -pname SSID -pvalue WiFi
+
+    @return Table which contains list of parameters and values otherwise false.
+--]]
+local function parse_args(args)
+    local ap_param_tbl = {}
+    local param_idx = 2
```

Comment:
It is _hidden function knowledge about call context_ that it's needed to skip first argument
By my opinion you should pass into function
* index of first parsing argument: ```parse_args(2)```
* table of arguments to parse:
```Lua
table.remove(args, 1)
parse_args(args)
```
(but i'm not about 2nd variant correctness in view of Lua best practice view)

Reply:
Current solution works well.